vscode-dotnet-runtime-library/src/index.ts

```diff
@@ -8,16 +8,19 @@ export * from './test/mocks/MockWindowDisplayWorker';
 export * from './test/mocks/MockEnvironmentVariableCollection';
 export * from './IExtensionContext';
 export * from './IDotnetAcquireContext';
+export * from './IDotnetListVersionsContext';
 export * from './IDotnetUninstallContext';
 export * from './IDotnetAcquireResult';
 export * from './IDotnetEnsureDependenciesContext';
 export * from './IExtensionContext';
 export * from './EventStream/EventStreamRegistration';
+export * from './EventStream/IWindowDisplayWorker';

```

Comment:
This is being exported here and imported in the sdk extensions's `extension.ts` but I don't see it ever being used. If that's the case, we should avoid exporting it from this library.

Reply:
This is used by the existing setPathEnvVar ;)